From: Kan-Ru Chen <koster@debian.org>
Date: Mon, 25 Sep 2017 01:07:58 +0800
Subject: Sort files in static library to make the build reproducible

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 114a6eb..1483a95 100644
--- a/Makefile
+++ b/Makefile
@@ -73,7 +73,7 @@ endif
 
 CC_CMD = $(QUIET_CC) $(CC) $(CFLAGS) -o $@ -c $<
 CXX_CMD = $(QUIET_CXX) $(CXX) $(CFLAGS) -o $@ -c $<
-AR_CMD = $(QUIET_AR) $(AR) cr $@ $^
+AR_CMD = $(QUIET_AR) $(AR) cr $@ `LC_ALL=C ls $^ 2>/dev/null`
 LINK_CMD = $(QUIET_LINK) $(CC) $(LDFLAGS) -o $@ $^ $(LIBS)
 MKDIR_CMD = $(QUIET_MKDIR) mkdir -p $@
 RM_CMD = $(QUIET_RM) rm -f $@
