From: Kan-Ru Chen <koster@debian.org>
Date: Mon, 25 Sep 2017 01:07:58 +0800
Subject: Sort files in static library to make the build reproducible

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 814a592..58698dc 100644
--- a/Makefile
+++ b/Makefile
@@ -53,7 +53,7 @@ endif
 MKTGTDIR = mkdir -p $(dir $@)
 CC_CMD = $(QUIET_CC) $(MKTGTDIR) ; $(CC) $(CFLAGS) -MMD -MP -o $@ -c $<
 CXX_CMD = $(QUIET_CXX) $(MKTGTDIR) ; $(CXX) $(CFLAGS) -MMD -MP -o $@ -c $<
-AR_CMD = $(QUIET_AR) $(MKTGTDIR) ; $(AR) cr $@ $^
+AR_CMD = $(QUIET_AR) $(MKTGTDIR) ; $(AR) cr $@ `LC_ALL=C ls $^ 2>/dev/null`
 ifdef RANLIB
   RANLIB_CMD = $(QUIET_RANLIB) $(RANLIB) $@
 endif
