From: =?utf-8?q?=22Kan-Ru_Chen_=28=E9=99=B3=E4=BE=83=E5=A6=82=29=22?=
 <koster@debian.org>
Date: Thu, 7 Jul 2016 00:21:18 +0800
Subject: Fix build with system gl and glfw

---
 Makerules | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Makerules b/Makerules
index a93b9cf..9c5f2a4 100644
--- a/Makerules
+++ b/Makerules
@@ -110,6 +110,11 @@ SYS_OPENJPEG_LIBS = -lopenjp2
 SYS_JBIG2DEC_LIBS = -ljbig2dec
 SYS_JPEG_LIBS = -ljpeg
 SYS_ZLIB_LIBS = -lz
+ifeq "$(shell pkg-config --exists gl glfw3 && echo yes)" "yes"
+HAVE_GLFW ?= yes
+SYS_GLFW_CFLAGS = $(shell pkg-config --cflags gl glfw3)
+SYS_GLFW_LIBS = $(shell pkg-config --libs gl glfw3)
+endif
 
 endif
 
