From: Kan-Ru Chen <kanru@kanru.info>
Date: Mon, 3 Sep 2018 08:53:44 +0900
Subject: MuPDF crossbuild use host cc for utils

---
 Makefile  | 4 ++++
 Makerules | 1 +
 2 files changed, 5 insertions(+)

diff --git a/Makefile b/Makefile
index 98e6567..3002bb5 100644
--- a/Makefile
+++ b/Makefile
@@ -180,6 +180,8 @@ $(SIGNATURE_OBJ) : $(PKCS7_HDR)
 # --- Generated PDF name tables ---
 
 NAMEDUMP_EXE := $(OUT)/scripts/namedump.exe
+$(NAMEDUMP_EXE): scripts/namedump.c | $(ALL_DIR)
+	$(QUIET_CC) $(CC_FOR_BUILD) $(CFLAGS) $< -o $@
 
 include/mupdf/pdf.h : include/mupdf/pdf/name-table.h
 NAME_GEN := include/mupdf/pdf/name-table.h source/pdf/pdf-name-table.h
@@ -197,6 +199,8 @@ generate: $(NAME_GEN)
 # --- Generated embedded font files ---
 
 HEXDUMP_EXE := $(OUT)/scripts/hexdump.exe
+$(HEXDUMP_EXE): scripts/hexdump.c | $(ALL_DIR)
+	$(QUIET_CC) $(CC_FOR_BUILD) $(CFLAGS) -o $@ $<
 
 FONT_BIN_DROID := $(sort $(wildcard resources/fonts/droid/*.ttf))
 FONT_BIN_NOTO := $(sort $(wildcard resources/fonts/noto/*.ttf))
diff --git a/Makerules b/Makerules
index 2e5bbce..fed3ce6 100644
--- a/Makerules
+++ b/Makerules
@@ -7,6 +7,7 @@ OS := $(OS:Windows_NT=MINGW)
 OS := $(OS:Darwin=MACOS)
 
 PKG_CONFIG ?= pkg-config
+CC_FOR_BUILD ?= $(CC)
 
 HAVE_LIBDL ?= yes
 
