Index: mupdf-1.5/platform/x11/pdfapp.h
===================================================================
--- mupdf-1.5.orig/platform/x11/pdfapp.h
+++ mupdf-1.5/platform/x11/pdfapp.h
@@ -12,7 +12,7 @@
 
 /* 25% .. 400% */
 #define MINRES 18
-#define MAXRES 288
+#define MAXRES 576
 
 typedef struct pdfapp_s pdfapp_t;
 
Index: mupdf-1.5/platform/x11/pdfapp.c
===================================================================
--- mupdf-1.5.orig/platform/x11/pdfapp.c
+++ mupdf-1.5/platform/x11/pdfapp.c
@@ -28,7 +28,7 @@ enum
 static void pdfapp_showpage(pdfapp_t *app, int loadpage, int drawpage, int repaint, int transition, int searching);
 static void pdfapp_updatepage(pdfapp_t *app);
 
-static const int zoomlist[] = { 18, 24, 36, 54, 72, 96, 120, 144, 180, 216, 288 };
+static const int zoomlist[] = { 18, 24, 36, 54, 72, 96, 120, 144, 180, 216, 288, 360, 480, 576 };
 
 static int zoom_in(int oldres)
 {
