From: Tor Andersson <tor.andersson@artifex.com>
Date: Tue, 8 Jan 2019 11:44:59 +0100
Subject: [PATCH] Bug 700442: Add a recursion depth check to prevent infinite
 recursion.

Origin: https://bugs.ghostscript.com/show_bug.cgi?id=700442
---
 source/svg/svg-run.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/source/svg/svg-run.c b/source/svg/svg-run.c
index 9429786..a117e2b 100644
--- a/source/svg/svg-run.c
+++ b/source/svg/svg-run.c
@@ -10,12 +10,15 @@
 #define DEF_HEIGHT 792
 #define DEF_FONTSIZE 12
 
+#define MAX_USE_DEPTH 100
+
 typedef struct svg_state_s svg_state;
 
 struct svg_state_s
 {
 	fz_matrix transform;
 	fz_stroke_state stroke;
+	int use_depth;
 
 	float viewport_w, viewport_h;
 	float viewbox_w, viewbox_h, viewbox_size;
@@ -1032,6 +1035,12 @@ svg_run_use(fz_context *ctx, fz_device *dev, svg_document *doc, fz_xml *root, co
 	float x = 0;
 	float y = 0;
 
+	if (++local_state.use_depth > MAX_USE_DEPTH)
+	{
+		fz_warn(ctx, "svg: too much recursion");
+		return;
+	}
+
 	svg_parse_common(ctx, doc, root, &local_state);
 	if (x_att) x = svg_parse_length(x_att, local_state.viewbox_w, local_state.fontsize);
 	if (y_att) y = svg_parse_length(y_att, local_state.viewbox_h, local_state.fontsize);
@@ -1164,6 +1173,7 @@ svg_run_document(fz_context *ctx, svg_document *doc, fz_xml *root, fz_device *de
 	/* Initial graphics state */
 	state.transform = ctm;
 	state.stroke = fz_default_stroke_state;
+	state.use_depth = 0;
 
 	state.viewport_w = DEF_WIDTH;
 	state.viewport_h = DEF_HEIGHT;
