From: Bastian Germann <bastiangermann@fishpost.de>
Date: Fri, 10 Apr 2020 18:05:55 +0200
Subject: Prevent thirdparty archive build

---
 Makefile |    1 -
 1 file changed, 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -217,7 +217,6 @@ MUPDF_LIB = $(OUT)/libmupdf.$(SO)
 $(MUPDF_LIB) : $(MUPDF_OBJ) $(THIRD_OBJ) $(THREAD_OBJ) $(PKCS7_OBJ)
 else
 MUPDF_LIB = $(OUT)/libmupdf.a
-THIRD_LIB = $(OUT)/libmupdf-third.a
 THREAD_LIB = $(OUT)/libmupdf-threads.a
 PKCS7_LIB = $(OUT)/libmupdf-pkcs7.a
 
