---
 Makefile         |    2 +-
 Makerules        |    2 +-
 fitz/image_jpx.c |    3 +++
 3 files changed, 5 insertions(+), 2 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -212,7 +212,7 @@
 	install -d $(DESTDIR)$(bindir) $(DESTDIR)$(libdir) $(DESTDIR)$(incdir) $(DESTDIR)$(mandir)/man1
 	install $(FITZ_LIB) $(DESTDIR)$(libdir)
 	install fitz/memento.h fitz/fitz.h pdf/mupdf.h xps/muxps.h cbz/mucbz.h $(DESTDIR)$(incdir)
-	install $(MUVIEW) $(MUDRAW) $(MUBUSY) $(DESTDIR)$(bindir)
+	install $(MUVIEW) $(MUDRAW) $(MUTOOL) $(DESTDIR)$(bindir)
 	install $(wildcard apps/man/*.1) $(DESTDIR)$(mandir)/man1
 
 # --- Clean and Default ---
--- a/Makerules
+++ b/Makerules
@@ -25,7 +25,7 @@
 
 ifeq "$(OS)" "Linux"
 SYS_FREETYPE_INC := `pkg-config --cflags freetype2`
-SYS_OPENJPEG_INC := `pkg-config --cflags libopenjpeg`
+SYS_OPENJPEG_INC := `if pkg-config --exists libopenjpeg1; then pkg-config --cflags libopenjpeg1; else echo "-L/usr/include"; fi`
 X11_LIBS := -lX11 -lXext
 endif
 
--- a/fitz/image_jpx.c
+++ b/fitz/image_jpx.c
@@ -50,8 +50,11 @@
 	evtmgr.info_handler = fz_opj_info_callback;
 
 	opj_set_default_decoder_parameters(&params);
+#ifdef OPJ_DPARAMETERS_IGNORE_PCLR_CMAP_CDEF_FLAG
+	/* Only openjpeg 1.5 support this */
 	if (indexed)
 		params.flags |= OPJ_DPARAMETERS_IGNORE_PCLR_CMAP_CDEF_FLAG;
+#endif
 
 	info = opj_create_decompress(format);
 	opj_set_event_mgr((opj_common_ptr)info, &evtmgr, ctx);
