---
 Makethird |    9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

--- a/Makethird
+++ b/Makethird
@@ -23,6 +23,10 @@
 V8_DIR := $(wildcard thirdparty/v8*)
 
 ifneq "$(V8_DIR)" ""
+HAVE_V8 := yes
+endif
+
+ifeq "$(HAVE_V8)" "yes"
 
 ifeq "$(build)" "release"
 V8_LIB_CHOICE := release
@@ -35,7 +39,9 @@
 V8_ARCH ?= x64
 
 # If you are building for 32bit linux use V8_ARCH=ia32
-ifeq "$(OS)" "Darwin"
+ifeq "$(OS)" "Linux"
+V8_LIBS := -lv8 -lpthread -ldl -lstdc++
+else ifeq "$(OS)" "Darwin"
 V8_LIBS := -L$(V8_DIR)/out-mac/$(V8_ARCH).$(V8_LIB_CHOICE)/ -lv8_base -lv8_snapshot -lstdc++
 else
 V8_LIBS := -L$(V8_DIR)/out/$(V8_ARCH).$(V8_LIB_CHOICE)/obj.target/tools/gyp -lv8_base -lv8_snapshot -lpthread -ldl -lstdc++
@@ -43,7 +49,6 @@
 
 V8_CFLAGS := -I$(V8_DIR)/include
 
-HAVE_V8 := yes
 endif
 
 # --- MuJS ---
